File: CodingInterview/CodingInterview/csharp_tests/test_7_prime_number.cs
using System;
using NUnit.Framework;

// Task 7
// Write a program to check if a number is a prime number
namespace CodingInterview
{
    public class Class7
    {
        public static bool Is_Prime(int number)
        {
            var prime_status = true;
            for(var k = 2;k < Math.Round((decimal)number / 2); k++)
            {
                if(number % k == 0)
                {
                    prime_status = false;
                    break;
                }
            }
            return prime_status;
        }

        [Test]
        [TestCase(13, true)]
        [TestCase(2000, false)]
        public void CheckIsPrime(int input, bool expected_value)
        {
            Assert.AreEqual(Is_Prime(input), expected_value);
        }
    }
}

File: python_tests/test_7_prime_number.py
# Task 7
# Write a program to check if a number is a prime number


def is_prime(number):
    prime_status = True
    for k in range(2, round(number / 2)):
        if number % k == 0:
            prime_status = False
            break
    return prime_status


def test_prime_number():
    assert is_prime(13) == True


def test_not_prime_number():
    assert is_prime(2000) == False
